trigger/task.go

```diff
@@ -355,10 +355,10 @@ func (t *taskTriggers) generate(sub *event.Subscription, pastTenseOverride, test
 					return nil, errors.Wrapf(err, "error getting execution task")
 				}
 				if executionTask.Details.Status == evergreen.TaskFailed {
-					if executionTask.Details.Description == evergreen.TaskDescriptionStranded {
-						shouldSkipTicket = true
-					} else {
-						shouldSkipTicket = false
+					shouldSkipTicket := executionTask.Details.Description == evergreen.TaskDescriptionStranded ||
+						executionTask.Details.Type == evergreen.CommandTypeSetup ||
+						executionTask.IsSystemUnresponsive()
```

Comment:
I don't think we need to check IsSystemUnresponsive; checking the details type should be good enough to determine if the display status was set to setup-failed.

Reply:
IsSystemUnresponsive checks a different symptom; i recommended that he add it here https://github.com/evergreen-ci/evergreen/pull/5243#discussion_r762282172